lld/COFF/InputFiles.h

```diff
@@ -362,6 +362,10 @@ class ImportFile : public InputFile {
   const coff_import_header *hdr;
   Chunk *location = nullptr;
 
+  // Auxiliary IAT symbol and chunk on ARM64EC.
+  DefinedImportData *impECSym = nullptr;
+  Chunk *auxLocation = nullptr;
```

Comment:
I was about to say that I don't see anything that sets `auxLocation` within this patch, but I see that we're passing it by reference to `DefinedImportData`, so I guess it gets set that way?

Reply:
Yes, it's filled with `setLocation` in `IdataContents::create`, like other `DefinedImportData` symbols.